packages/test/snapshots/src/messageSchema.ts

```diff
@@ -0,0 +1,433 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { MergeTreeDeltaType } from "@fluidframework/merge-tree";
+
+export const joinContentsSchema = {
+    type: "null",
+};
+
+export const joinDataSchema = {
```

Comment:
Some of these schema were difficult to get from the code and involved examination of a real world document... Might be worthwhile for anyone that writes messages to make this more clear where it's not already?